CMakeLists.txt

```diff
@@ -451,7 +451,7 @@ set_target_properties(irods_filesystem_server PROPERTIES POSITION_INDEPENDENT_CO
 add_library(
   irods_user_administration_client
   OBJECT
-  ${CMAKE_CURRENT_SOURCE_DIR}/lib/core/src/user_administration.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/lib/administration/src/user_administration.cpp
```

Comment:
There are headers specific to the user administration library in lib/.

Consider moving those as well.

The files of interest are:
- lib/core/include/user.hpp
- lib/core/include/group.hpp
- lib/core/src/user.cpp
- lib/core/src/group.cpp

Reply:
Good idea.